integration-tests/src/main/java/org/apache/druid/testing/clients/CoordinatorResourceTestClient.java

```diff
@@ -241,6 +241,7 @@ public HttpResponseStatus getProxiedOverlordScalingResponseStatus()
       );
     }
 
+    @SuppressWarnings("unused") // It's unclear whether this call to readValue() has desirable side effects.
```

Comment:
I don't think it has any meaningful side effects. `response.getContent()` is a string that's already been read, so it doesn't trigger reading a stream or anything like that. Should be fine to remove.

Reply:
Removed